lib/kamal/cli/templates/deploy.yml

```diff
@@ -18,6 +18,8 @@ servers:
 proxy:
   ssl: true
   host: app.example.com
+  # Change the default application port from 80 to 3000. Useful for non-Thruster apps.
```

Comment:
These ports are generic and not Thruster-related/dependent in any way.

```suggestion
  # Change the default application port from 80 to 3000.
```

Reply:
I meant to say that with Thruster you will have port 80 exposed in container by default, without Thruster your container will expose 3000.